.cypress/integration/3_panels.spec.ts

```diff
@@ -10,578 +10,624 @@ import {
-      cy.get('.euiTableRow').should('have.length', 1);
+      it('Deletes the panel', () => {
+        createSavedObjectPanel();
+        cy.get('a[data-test-subj="breadcrumb last"]').click();//refresh so panel appears
+        cy.get('input[data-test-subj="checkboxSelectAll"]').click();
+        openActionsDropdown();
+        cy.get('button[data-test-subj="deleteContextMenuItem"]').click({ force: true });
+        cy.get('button[data-test-subj="popoverModal__deleteButton"]').should('be.disabled');
+        cy.get('input.euiFieldText[placeholder="delete"]').focus().type('delete');
+        cy.get('button[data-test-subj="popoverModal__deleteButton"]').should('not.be.disabled');
+        cy.get('button[data-test-subj="popoverModal__deleteButton"]').click({ force: true });
+        cy.get('h2[data-test-subj="customPanels__noPanelsHome"]').should('exist');
+      });
 
-      // Search for teh Saved Object panel
-      cy.get('[aria-label="Clear input"]').click();
-      cy.get('input[data-test-subj="operationalPanelSearchBar"]').focus().type('Saved Object', {
-        delay: 50,
+      it('Redirects to observability dashboard from OSD dashboards', () => {
+        moveToOsdDashboards();
+        cy.location('pathname').should('eq', '/app/dashboards');
+        cy.get('[data-test-subj="dashboardListingTitleLink-Test-Panel"]').click();
+        cy.location('pathname').should('eq', '/app/observability-dashboards');
+      });
+
+      it('Redirects to observability dashboard from OSD dashboards with edit', () => {
+        moveToOsdDashboards();
+        cy.location('pathname').should('eq', '/app/dashboards');
+        cy.get('[data-test-subj="dashboardListingTitleLink-Test-Panel"]')
+          .closest('tr')
+          .get('span.euiToolTipAnchor > button.euiButtonIcon')
+          .eq(0)
+          .click();
+        cy.location('pathname').should('eq', '/app/observability-dashboards');
+        cy.location('hash').should('include', '/edit');
       });
 
-      cy.get('a.euiLink').contains('Saved Object').should('exist');
-      cy.get('.euiTableRow').should('have.length', 1);
+      it('Redirects to observability dashboard from OSD dashboards with create', () => {
+        moveToOsdDashboards();
+        cy.location('pathname')
+          .should('eq', '/app/dashboards')
+          .then(() => {
+            cy.get('div#createMenuPopover').click();
+          });
+        cy.get('[data-test-subj="contextMenuItem-observability-panel"]').click();
+        cy.location('pathname').should('eq', '/app/observability-dashboards');
+        cy.location('hash').should('include', '/create');
+      });
     });
   });
 
-  describe('with a SavedObjects Panel', () => {
+  describe('Testing a panel', () => {
     beforeEach(() => {
-      createSavedObjectPanel();
-      moveToPanelHome();
-      cy.get('.euiTableRow').should('have.length', 1);
+      const test_name = `test_${new Date().getTime()}`;
+      createSavedObjectPanel(test_name).as('thePanel');
```

Comment:
Each tested feature of a panel should be independent and exclusive to the intended functionality.